### PR TITLE
Add basic install test

### PR DESCRIPTION
**Pull request by DevonianTeuchter**
Add a basic install test that does some package installation, upgrade and removal.

### CODE REVIEW

File: tests/zopen_check_basic_install [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  fail "Wrong which was used: expected: '${zopenenv}/usr/local/bin/which'; actual: '${whichWhich}'"
+fi
+
+echo "Running basic upgrade test"
+if ! zopen upgrade; then
+  fail "Basic upgrade test failed. See previous errors"
+fi
+
+echo "Testing removal"
+if ! zopen remove which; then
```
Comment (IgorTodorovskiIBM) [inline tests/zopen_check_basic_install:58]: Should we another `zopen remove which` test where we expect it to fail because it's no longer installed?

Comment (DevonianTeuchter) [reply tests/zopen_check_basic_install:58]: Would make sense - let me add it

Comment (netskink) [reply tests/zopen_check_basic_install:58]: fwiw, I added some tests for the patch I did on zopen install findutilsport addressing my recent changes.

**Review by IgorTodorovskiIBM — APPROVED** [review]
LGTM

**Comment by netskink** [comment]
Excellent !many thanks!

**Pull request merged**